### PR TITLE
(DE27306) Navigating back from upcoming assignments overlay NO LONGER changes the widget time period

### PR DESCRIPTION
**Pull request by trevorjonesd2l**

### CODE REVIEW

File: test/components/d2l-upcoming-assessments.js [hunk 1]
```diff
@@ -86,7 +86,7 @@ describe('<d2l-upcoming-assessments>', function() {
 		describe('_getActivityInfo', function() {
 
 			it('sets the nextPeriodUrl if the link exists on the provided activities object', function() {
-				activities.getLinkByRel = sandbox.stub().withArgs('https://activities.api.brightspace.com/rels/next-period').returns({ href: nextPeriodUrl });
+				activities.getLinkByRel = sandbox.stub().withArgs('https://act;ivities.api.brightspace.com/rels/next-period').returns({ href: nextPeriodUrl });
```
Comment (CodeBaboon) [inline test/components/d2l-upcoming-assessments.js]: Was this change intentional or just an accident?

Comment (trevorjonesd2l) [reply test/components/d2l-upcoming-assessments.js]: Accidental :p Weird it didn't break anything! I'll fix

**Comment by trevorjonesd2l** [comment]
New PR because rebase killed me 💀
